src/Nethermind/Ethereum.Blockchain.Test/MetaTests.cs

```diff
@@ -40,7 +40,7 @@ public void All_categories_are_tested()
             {
                 string expectedTypeName = ExpectedTypeName(directory);
                 Type type = types.SingleOrDefault(t => string.Equals(t.Name, expectedTypeName, StringComparison.InvariantCultureIgnoreCase));
-                if(type == null && directory != "stEWASMTests" && directory != "runtimes")
+                if(type == null && directory != "stEWASMTests" && directory != "Specs" && directory != "runtimes")
```

Comment:
this was updated after linking the latest ethereum/tests after a submodules recursive update